Custom Field Constraints/CFConstraints.quick.add.js

```diff
@@ -122,6 +122,13 @@ tau.mashups
             _createCFDataBindConfig: function(cf) {
                 return {
                     caption: cf.name,
+                    config: {

```

Comment:
This bug was found due investigation, so I included to fix because it is critical. Quick ad doesn't work on axis value with constraints